src/Whmcs/WhmcsConfig.php

```diff
@@ -30,6 +33,9 @@ class WhmcsConfig
      */
     protected $countOptions = self::PRE_INSTALL;
 
+    const MEM_BILLING_DESC = 'Billing ID of the RAM. Setting this will override user-selected RAM options. ex: mem-1 | 8 GB RAM';
```

Comment:
text will need to be updated per above comments about precedence. This should also include something to the effect of "this field does nothing when the Memory Configurable Option is present".